core/src/components/input/test/item/input.e2e.ts

```diff
@@ -0,0 +1,71 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('input: item'), () => {
+    test('should render correctly in list with no fill', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-list>
+          <ion-item>
+            <ion-input
+              label="Email"
+              value="hi@ionic.io"
+              helper-text="Enter your email"
+              maxlength="20"
+              counter="true"
+            ></ion-input>
+          </ion-item>
+        </ion-list>
+      `,
+        config
+      );
+      const list = page.locator('ion-list');
+      expect(await list.screenshot()).toMatchSnapshot(screenshot(`input-list-no-fill`));
+    });
+    test('should render correctly in inset list with no fill', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-list inset="true">
+          <ion-item>
+            <ion-input
+              label="Email"
+              value="hi@ionic.io"
+              helper-text="Enter your email"
+              maxlength="20"
+              counter="true"
+            ></ion-input>
+          </ion-item>
+        </ion-list>
+      `,
+        config
+      );
+      const list = page.locator('ion-list');
+      expect(await list.screenshot()).toMatchSnapshot(screenshot(`input-inset-list-no-fill`));
+    });
+    test('input renders as modern when label is set asynchronously', async ({ page }, testInfo) => {
```

Comment:
Optional: Looks like this one could be pulled into a separate block and only run on LTR/one mode.

Reply:
I ended up making this a spec test. Good catch!